composer.json

```diff
@@ -5,25 +5,25 @@
   "type": "library",
   "minimum-stability": "dev",
   "require": {
-    "php": ">=7.1",
+    "php": ">=7.4",
     "ext-json": "*",
     "google/apiclient": "^1.1",
     "ezyang/htmlpurifier": "^4.12",
     "dompdf/dompdf": "^0.8.0",
     "tedivm/jshrink": "1.1.0",
     "vlucas/phpdotenv": "^v5.4",
-    "fzaninotto/faker": "^1.9.1",
+    "fzaninotto/faker": "dev-master",
```

Comment:
Waarom moet dit `dev-master` zijn?

Reply:
Omdat ze PHP 8 compatibility hebben toegevoegd, maar daar geen tag voor hebben gepubliceerd.

Reply:
Interessant. Bij erfgoedkit.nl is de version constraint `^1.9.1`. Leek mij al sterk dat Laravel een `dev-master` requirement heeft, maar onderwater wordt gewoon `dev-master` gebruikt. Dat staat in de lock file. 🤷‍♂️